internal/binutils/binutils_test.go

```diff
@@ -401,3 +414,125 @@ func TestOpenMalformedMachO(t *testing.T) {
 		t.Errorf("Open: got %v, want error containing 'Mach-O'", err)
 	}
 }
+
+func TestObjdumpVersionChecks(t *testing.T) {
+	// Test that the objdump version strings are parsed properly.
+	type objdumpTest struct {
+		desc string
+		os   string
+		ver  string
+		want bool
+	}
+
+	for _, tc := range []objdumpTest{
+		{
+			// Valid Apple LLVM version string with usable version.
```

Comment:
Here and below - no need for this comment, the `desc` field covers this.

Reply:
Done.